---
source: src/validation/tests/generic_validation_tests.rs
expression: make_readable(&diagnostics)
---
SyntaxError { message: "Invalid type nature for generic argument. DATE_AND_TIME is no Num.", range: [SourceRange { range: 135..136 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. DATE_AND_TIME is no Num.", range: [SourceRange { range: 208..209 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. DATE is no Num.", range: [SourceRange { range: 276..277 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. TIME_OF_DAY is no Num.", range: [SourceRange { range: 349..350 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. TIME_OF_DAY is no Num.", range: [SourceRange { range: 417..418 }], err_no: type__invalid_nature }

